packages/mongo/mongo_connection.js

```diff
@@ -850,6 +891,35 @@ Object.assign(MongoConnection.prototype, {
     const { includeCollections, excludeCollections } = oplogOptions;
     if (firstHandle) {
       var matcher, sorter;
+      
+      // Check if Change Streams are available and enabled
+      const canUseChangeStreams = [
+        function () {
+          // Change Streams require MongoDB 3.6+ and replica set
+          return self._supportsChangeStreams && !ordered &&
+            !callbacks._testOnlyPollCallback;
+        },
+        function () {
+          // Check if change streams are explicitly disabled
+          const mongoSettings = Meteor.settings?.packages?.mongo || {};
+          return mongoSettings.useChangeStreams !== false;
```

Comment:
we should add this flag to the docs – I would change it to be default on for new projects, it would be nice to add it to our skeletons.

Maybe naming could be a string:


`reactivityMode` : `ChangeStream` | `Oplog`